packages/ui/components/dialog/dialog.stories.mdx

```diff
@@ -53,6 +53,11 @@ The `Dialog` component is composed of the following components:
 <Canvas>
   <Story
     name="Dialog"
+    parameters={{
+      nextjs: {
+        appDirectory: true,
+      },
+    }}
```

Comment:
specify that this component uses features from next.js 13+ imported from next/navigation